usr/iscsistart.c

```diff
@@ -254,7 +258,7 @@ static int login_session(struct node_rec *rec)
 	 * login.
 	 */
 	for (msec = 50; msec <= 15000; msec <<= 1) {
-		int tmo = ISCSID_REQ_TIMEOUT * 10;
+		int tmo = ISCSID_REQ_TIMEOUT * login_req_timeout;
```

Comment:
I don't like multiplying ISCSID_REQ_TIMEOUT times the login_req_timeout. It just so happens that currently ISCSID_REQ_TIMEOUT is one second, but what if it changes to 5 seconds? Then your iscsistart timout would multiply times 5! Can you just login_req_timeout not depend on ISCSID_REQ_TIMEOUT?